tools/installpsh-debian.sh

```diff
@@ -126,7 +126,15 @@ echo "*** Setting up PowerShell Core repo..."
 # Import the public repository GPG keys
 curl https://packages.microsoft.com/keys/microsoft.asc | $SUDO apt-key add -
 #Add the Repo
-curl https://packages.microsoft.com/config/ubuntu/$REV/prod.list | $SUDO tee /etc/apt/sources.list.d/microsoft.list
+case $REV in
+    8) $SUDO sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-jessie-prod jessie main" > /etc/apt/sources.list.d/microsoft.list' ;;
+    9) $SUDO sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" > /etc/apt/sources.list.d/microsoft.list' ;;
```

Comment:
This file is for all debian variants including Ubuntu.  Therefore, this change breaks CI.  This MUST be fixed.

Reply:
see test failure:
https://travis-ci.org/PowerShell/PowerShell/jobs/349373111#L519

Reply:
My bad, thought it was for Debian only. So, in order for it to work for every Debian-based distro, do we need to "case" every one of them? Something like
```
if distrib_id == ubuntu
  add_ubuntu_repo
elif distrib_id == linuxmint
  add_mint_repo
elif distrib_id == debian
  add_debian_repo
elif ...
```

I am afraid I don't have time to check for every possible distro :confused:.

But I can make sure it works for both Debian and Ubuntu to start with.

Reply:
Since it was only working for ubuntu before, keeping ubuntu support and adding debian works.  Just added a comment that support for other distros is needed.